resources/function-controller/templates/functions.crd.yaml

```diff
@@ -27,10 +25,8 @@ spec:
   group: serverless.kyma-project.io
   names:
     kind: Function
-    listKind: FunctionList
     plural: functions
-    singular: function
-  scope: Namespaced
+  scope: ""
```

Comment:
I think we are missing the `Namespaced` value for scope.

From documentation of kubernetes 1.16 - [CustomResourceDefinitionSpec](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.16/#customresourcedefinitionspec-v1beta1-apiextensions-k8s-io)

>  Allowed values are `Cluster` and `Namespaced`. Default is `Namespaced`.

Reply:
I copy it from our folder with component source